pkg/landscaper/gardenlet/applier/vpa_crd.go

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2020 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
```

Comment:
Same like above, please remove all of this. It doesn't belong into g/g.

Reply:
Please see my response above. Dont think we should just label it as a prerequisite and assume it is there. This only works for us, but the landscaper is also meant as community installer.